images/win/scripts/Installers/Install-Go.ps1

```diff
@@ -29,36 +29,40 @@ function Install-GoVersion
 
     # Extract the zip archive.  It contains a single directory named "go".
     Write-Host "Extracting Go $latestVersion..."
-    Expand-Archive -Path $goArchPath -DestinationPath "C:\" -Force
+    $toolDirectory = Join-Path $env:AGENT_TOOLSDIRECTORY "go\$latestVersion"
+    7z.exe x $goArchPath -o"$toolDirectory" -y | Out-Null
+
+    # Rename the extracted "go" directory to "x64" for full path "C:\hostedtoolcache\windows\Go\1.14.2\x64\..."
+    Rename-Item -path "$toolDirectory\go" -newName "x64"
+    $fullArchPath = "$toolDirectory\x64"
 
     # Delete unnecessary files to conserve space
     Write-Host "Cleaning directories of Go $latestVersion..."
-    if (Test-Path "C:\go\doc")
+    if (Test-Path "$fullArchPath\doc")
     {
-        Remove-Item -Recurse -Force "C:\go\doc"
+        Remove-Item -Recurse -Force "$fullArchPath\doc"
     }
-    if (Test-Path "C:\go\blog")
+    if (Test-Path "$fullArchPath\blog")
     {
-        Remove-Item -Recurse -Force "C:\go\blog"
+        Remove-Item -Recurse -Force "$fullArchPath\blog"
     }
 
-    # Rename the extracted "go" directory to include the Go version number (to support side-by-side versions of Go).
-    $newDirName = "Go$latestVersion"
-    Rename-Item -path "C:\go" -newName $newDirName
+    # Create symlink in old location
+    New-Item -Path "C:\go$latestVersion" -ItemType SymbolicLink -Value $fullArchPath
```

Comment:
isn't this missing a `\`?

Reply:
@alepauly , no, in old location Go was installed in folders called `go1.14.2`, `go1.13.2` and etc. I have tested symlinks on created VM manually, it works correct.